content/en/examples/admin/sched/my-scheduler.yaml

```diff
@@ -30,6 +30,22 @@ roleRef:
   name: system:volume-scheduler
   apiGroup: rbac.authorization.k8s.io
 ---
+
+apiVersion: v1
+data:
```

Comment:
nit: I'd put this after `kind` so it's more obvious that this is a ConfigMap being defined.